scripts/eurostat/health_determinants/fruits_vegetables/download_input_files.py

```diff
@@ -0,0 +1,96 @@
+    Args:
+        download_directory (str):Location where the files need to be downloaded.
+    Returns:
+        None
+    """
+    # List to provide the URLs of input files to download script.
+    INPUT_URLS = [
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+        "BulkDownloadListing?file=data/hlth_ehis_fv3e.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_fv3i.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_fv3u.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_fv1e.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_fv1u.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_fv3b.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_fv3c.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_fv3d.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_fv3m.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_fv1b.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_fv1c.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_fv1i.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_fv1m.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_fv1d.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_fv7e.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_fv7m.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_fv7i.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_de7.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_de8.tsv.gz",
+        "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+            "BulkDownloadListing?file=data/hlth_ehis_fv5e.tsv.gz"
+
+    ]
```

Comment:
INPUT_URLS =  ['https://ec.europa.eu....' + file for file in files]
where files is a list of files to be downloaded